Tests/Common/Securities/CashTests.cs

```diff
@@ -601,9 +601,101 @@ public void EnsureCurrencyDataFeedDoesNothingWithUnsupportedCurrency()
             Assert.IsEmpty(added);
         }
 
+        [TestCaseSource(nameof(cryptoBrokerageStableCoinCases))]
+        public void CryptoStableCoinMappingIsCorrect(IBrokerageModel brokerageModel, string accountCurrency, string stableCoin, bool shouldThrow, Symbol expectedConversionSymbol)
+        {
+            var cashBook = new CashBook() {AccountCurrency = accountCurrency};
+            var cash = new Cash(stableCoin, 10m, 1m);
+            cashBook.Add(cash.Symbol, cash);
+
+            var subscriptions = new SubscriptionManager();
+            var dataManager = new DataManagerStub(TimeKeeper);
+            subscriptions.SetDataManager(dataManager);
+            var abcConfig = subscriptions.Add(Symbols.SPY, Resolution.Minute, TimeZone, TimeZone);
+            var securities = new SecurityManager(TimeKeeper);
+            securities.Add(
+                Symbols.SPY,
```

Comment:
Cool tests!
Is adding SPY a requirement for the test, why? if not, let's just remove it 💯

Reply:
Dropped it :grin: